CorsixTH/Lua/dialogs/fullscreen.lua

```diff
@@ -75,28 +75,28 @@ function UIFullscreen:onChangeResolution()
   end
 end
 
-function UIFullscreen:draw(canvas, x, y)
+function UIFullscreen:draw(canvas, dx, dy)
```

Comment:
dx and dy typically express "difference x" or "difference y", rather use xpos, or x_pos, or something like that, as it expresses "position", much what x or y does.